src/components/MasterItems.js

```diff
@@ -73,21 +78,37 @@ class MasterItems extends Component {
     return this.state.entries.map((entry, i) => dom.tr(
       { className: 'entries', key: i },
       dom.td({ className: 'entry-name col-xs-5' }, dom.p(null, entry.name)),
-      dom.td({ className: 'entry-comment col-xs-5' }, dom.input({
-        className: `input form-control add-comment-${entry.name}`,
-        onChange: this.addComment.bind(null, i),
-        placeholder: 'Comment',
-        onKeyPress: this.handleKeyPress,
-        value: entry.comment
-      })),
-      dom.td({ className: 'entry-quantity col-xs-2' }, dom.input({
-        className: `input form-control add-quantity-${entry.name}`,
-        type: 'number',
-        max: '50',
-        onChange: this.setQuantity.bind(null, i),
-        onKeyPress: this.handleKeyPress,
-        value: entry.quantity
-      })),
+      dom.td(
+        { className: 'entry-comment col-xs-4' },
+        dom.input(
+          {
+            className: `input form-control add-comment-${entry.name}`,
+            onChange: this.addComment.bind(null, i),
+            placeholder: 'Comment',
+            onKeyPress: this.handleKeyPress,
+            value: entry.comment
+          }
+        )
+      ),
+      dom.td(
+        { className: 'entry-quantity col-xs-3 row' },
```

Comment:
Can we do bigger? It's showing up all messed up on the ipad.
![img_0054](https://cloud.githubusercontent.com/assets/1305229/12636542/9d368846-c542-11e5-9e8d-48b36316b98d.PNG)